rest/dao/src/main/resources/io/syndesis/dao/deployment.json

```diff
@@ -821,6 +891,69 @@
               }
             }]
           }
+        },
+        {
+          "id": "sql-stored-start-connector",
+          "name": "Periodic Stored Procedure Invocation",
+          "description": "Periodically Invoke a Stored Procedure to get Data out",
+          "tags": [ "dynamic" ],
+          "pattern": "From",
+          "actionType": "connector",
+          "descriptor": {
+            "camelConnectorGAV": "io.syndesis:sql-stored-start-connector:1.2-SNAPSHOT",
```

Comment:
How to deal with depending on a SNAPSHOTS? How does this work in the mono repo? Hardcoding for now..